pkgs/by-name/e1/e1s/package.nix

```
@@ -25,6 +25,6 @@ buildGoModule {
     changelog = "https://github.com/keidarcy/e1s/releases/tag/v${version}";
     license = licenses.mit;
     mainProgram = "e1s";
-    maintainers = with maintainers; [ zelkourban ];
+    maintainers = with maintainers; [ zelkourban carlossless ];
```

```suggestion
    maintainers = with maintainers; [
      zelkourban
      carlossless
    ];
```

Please fix the format using `nix fmt`

---

Thanks, addressed.